pkg/legacy/converter.go

```diff
@@ -195,6 +195,9 @@ func extractURLAPIKeys(agentConfig Config) error {
 
 	additionalEndpoints := map[string][]string{}
 	for idx, url := range urls {
+		if url == "" || keys[idx] == "" {
+			return fmt.Errorf("empty additional `dd_url' or 'api_key' found")
```

Comment:
I think we can just ignore it

Reply:
Make sense. Let's just be a little more explicit in the log message to help users fix the issue then